src/advanced/node-management.md

```diff
@@ -168,6 +168,33 @@ JSON object with the following fields:
 }
 ```
 
+### Healthcheck
+
+```none
+GET {system_base_path}/healthcheck
+```
+
+Returns a boolean representing if the node is connected to other peers.
+
+#### Parameters
+
+None.
+
+#### Response
+
+JSON object with the following fields:
+
+- **connectivity**: bool
```

Comment:
You need to add two spaces after `bool` in order to make a line-break, similar to how it's done in other endpoint descriptions.